include/zephyr/sys/sem.h

```diff
@@ -138,6 +138,66 @@ int sys_sem_take(struct sys_sem *sem, k_timeout_t timeout);
  */
 unsigned int sys_sem_count_get(struct sys_sem *sem);
 
+/**
```

Comment:
You might want to introduce this helper in a separate commit, so it gets more visibility, can be explained in the commit message and be attributed to the right subsys?

Reply:
Done!